dbc2val/requirements.txt

```diff
@@ -1,10 +1,95 @@
-pyserial
-cantools
-python-can
```

Comment:
Why is python-can removed? We have some `import can` in the code which seem to rely on this.

Reply:
It is still being pulled in as a transitive dependency of can-j1939 and cantools. So it seemed feasible to let these two deps determine the particular version that works for them.
However, I see your point that if our code explicitly depends on it, we should probably also explicitly define the dependency. In that case, should I derive the version (range) from the version that has been resolved to work for can-j1939 and cantools?

Reply:
I do not have a strong opinion. If we want to explicitly state it with version we can take what has been derived. But then maybe add a comment that we typically can use what works for can-j1939 and cantools. Or possibly specifying it without version and just state in a comment that version is implicitly selected by the other packages, or just mention it in a comment so that it is understandable why it isn't listed.

Reply:
I have added python-can to the *requirements.in* file.